src/views/view.rb

```diff
@@ -1,115 +1,119 @@
-class View
-  include Input
-  include Output
-
-  START_STATES = { create: :create_account,
-                   load: :load_account }.freeze
-
-  def initialize(state = :start)
-    @account_operations = AccountConnect.new
-    @state = state
-  end
-
-  def run
-    loop do
-      @state == :shutdown ? break : send(@state)
+module Banking
+  module View
+    class View
```

Comment:
src/view`s`/view.rb

Please, check the naming throughout the project

Reply:
fixed